app/assets/stylesheets/govuk_publishing_components/components/_input.scss

```diff
@@ -31,19 +31,45 @@
   }
 }
 
+.gem-c-input__prefix {
+  @include govuk-font($size: 19);
+
+  background-color: govuk-colour("light-grey", $legacy: "grey-3");
+  border: $govuk-border-width-form-element solid $govuk-input-border-colour;
+  box-sizing: border-box;
+  cursor: default; // emphasise non-editable status of prefixes and suffixes
+  display: inline-block;
+  flex: 0 0 0;
+  border-right: 0;
+  padding-left: govuk-spacing(3);
```

Comment:
@simonwhatley correct me if I'm wrong, but as I understand the expected outcome from a design perspective would require consistent padding (`govuk-spacing(1);`), as before, but limited to a minimum width equal to the height (which is 40px, or the equivalent value in rems).

This can be achieved with
```
  padding: govuk-spacing(1);
  min-width: 40px;
  @if $govuk-typography-use-rem {
      min-width: govuk-px-to-rem(40px);
  }
```

Reply:
Yes, that's my expectation @alex-ju. 👍

Reply:
I went with the weird padding arrangement over min-width because while it worked fine for the prefix example I couldn't seem to get it to work for the suffix. I tried having both a width setting of auto and a min-width setting of 40px, but from reading the css docs it seems that min-width always overrides width anyway. Please let me know if i'm missing something obvious. 

<img width="378" alt="Screenshot 2020-05-21 at 09 33 28" src="https://user-images.githubusercontent.com/41922771/82539938-2f315680-9b46-11ea-89fa-4a538d57a81a.png">
<img width="313" alt="Screenshot 2020-05-21 at 09 33 32" src="https://user-images.githubusercontent.com/41922771/82539942-2fc9ed00-9b46-11ea-8f56-e9446bb319b5.png">

Reply:
It seems the above problem was being caused by the flex: 0 0 0; on line 42. It was not allowing the box to grow with the text, instead it was imposing a width limit of 40px.